### ISSUE TITLE
Module Crash after reload

### ISSUE THREAD

**Issue by bryce-seifert** [bitfocus/companion-module-bmd-videohub#30]
On Companion rc-1, I'm getting constant crashes after reloading the videohub module.

Here is the error being emitted
```
Instance "videohub_3" failed to init: Error: Call timed out Error: Call timed out
    at IpcWrapper.sendWithCb (/Users/bs29698/Documents/GitHub/companion/node_modules/@companion-module/base/dist/host-api/ipc-wrapper.js:35:30)
    at SocketEventsHandler.init (file:///Users/bs29698/Documents/GitHub/companion/lib/Instance/Wrapper.js:72:37)
    at Monitor.initHandler (file:///Users/bs29698/Documents/GitHub/companion/lib/Instance/Host.js:147:7)
    at Monitor.emit (node:events:513:28)
    at Monitor.emit (node:domain:489:12)
    at ChildProcess.<anonymous> (/Users/bs29698/Documents/GitHub/companion/vendor/respawn/index.js:142:10)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess.emit (node:domain:489:12)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
```

This seems to happen with our Videohub 288, but not with our Videohub 72 for some reason.

Also, when you first add the module it connected. It's only after reloading the module in the web UI, or restarting companion that is crashes

**Comment by Julusian** [comment]
duplicate of #31 

I think I have a fix
